tests/models_tests/models_test.go

```diff
@@ -308,8 +313,9 @@ func TestRetrieveBreakdownStatsBySourceIdAndCohortIdWithResultsWithOnePersonTwoH
 		totalPersonInsecondLargestCohortWithValue += statLargeCohort.NpersonsInCohortWithValue
 	}
 
-	if totalPersonInsecondLargestCohortWithValue != secondLargestCohort.CohortSize+1 {
-		t.Errorf("Expected total peope in return data to be 1 larger than cohort size, but total people was %d and cohort size is %d", totalPersonInsecondLargestCohortWithValue, secondLargestCohort.CohortSize)
+	expectedWithValueInSecondLargest := secondLargestCohort.CohortSize - 1 // because 2nd largest has one person that has a NULL HARE entry...
```

Comment:
this change is also related to the fact that I removed one entry from secondLargestCohort in `tests/setup_local_db/test_data_results_and_cdm.sql`, so secondLargestCohort only has one person remaining w/ a double HARE value (person id = 2 has `ASN` and `AFR`)